mypy/util.py

```diff
@@ -753,7 +753,7 @@ def format_error(
 
 def is_typeshed_file(file: str) -> bool:
     # gross, but no other clear way to tell
-    return 'typeshed' in os.path.abspath(file).split(os.sep)
+    return 'basedtypeshed' in os.path.abspath(file).split(os.sep)
```

Comment:
Do you think supporting both is better?

Reply:
not if we intend to make breaking changes that are incompatible with cringetypeshed